cmd/bucket-replication.go

```diff
@@ -2980,7 +2993,7 @@ func (p *ReplicationPool) saveMRFEntries(ctx context.Context, entries map[string
 		return err
 	}
 
-	configFile := path.Join(replicationMRFDir, mustGetUUID()+".bin")
+	configFile := path.Join(replicationMRFDir, globalLocalNodeNameHex+".bin")
```

Comment:
if this is per node, do we need to erasure code it?

Reply:
this won't scale if we have 600+ nodes.

Reply:
@poornas ^^?

Reply:
why is it a problem if this file is erasure coded? we are not trying to persist mrf at the time of cluster shutdown

Reply:
changed this